### PR TITLE
Update node version to v14

### PR DESCRIPTION
**Pull request by TheIronDev**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 - Try to extract context.params from triggered data [pull request](https://github.com/firebase/firebase-functions-test/pull/114)
 - Implement firebase-functions-test wrapV2 [pull request](https://github.com/firebase/firebase-functions-test/pull/131)
+- Update node version to v14 [pull request](https://github.com/firebase/firebase-functions-test/pull/135)
```
Comment (bkendall) [inline CHANGELOG.md:3]:
```suggestion
- Update minimum Node version to v14 [pull request](https://github.com/firebase/firebase-functions-test/pull/135)
```

File: package-lock.json [hunk 2]
```diff
@@ -1,8298 +1,8 @@
 {
   "name": "firebase-functions-test",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```
Comment (bkendall) [inline package-lock.json:4]: This seems like a regression - you may want to regenerate this with npm 8

**Review by bkendall — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 2 above)
Comment (TheIronDev) [reply package-lock.json]: ah yeah I was jumping around node versions a bit. Updated!

Comment (TheIronDev) [reply package-lock.json]: And by updated I mean re-generated.

**Review by TheIronDev — COMMENTED** [review]
LGTM

**Pull request merged**